# Tag id
int8 id

# Tag center in pixels [u,v]
float64[] center

# Tag corners in pixels [u,v] start at bottom left (bl), continue CCW to top left (tl)
float64[] corner_bl
float64[] corner_br
float64[] corner_tr
float64[] corner_tl